GraknConsole.java

```diff
@@ -210,45 +220,62 @@ private void runQuery(GraknClient.Transaction tx, String queryString) {
                 printer.info("Concepts have been undefined");
             } else if (query instanceof GraqlInsert) {
                 Stream<ConceptMap> result = tx.query().insert(query.asInsert());
-                printCancellableResult(result, x -> printer.conceptMap(x, tx));
+                printCancellableResult(executorService, result, x -> printer.conceptMap(x, tx));
             } else if (query instanceof GraqlDelete) {
                 tx.query().delete(query.asDelete()).get();
                 printer.info("Concepts have been deleted");
             } else if (query instanceof GraqlMatch) {
                 Stream<ConceptMap> result = tx.query().match(query.asMatch());
-                printCancellableResult(result, x -> printer.conceptMap(x, tx));
+                printCancellableResult(executorService, result, x -> printer.conceptMap(x, tx));
             } else if (query instanceof GraqlMatch.Aggregate) {
                 Numeric answer = tx.query().match(query.asMatchAggregate()).get();
                 printer.numeric(answer);
             } else if (query instanceof GraqlMatch.Group) {
                 Stream<ConceptMapGroup> result = tx.query().match(query.asMatchGroup());
-                printCancellableResult(result, x -> printer.conceptMapGroup(x, tx));
+                printCancellableResult(executorService, result, x -> printer.conceptMapGroup(x, tx));
             } else if (query instanceof GraqlMatch.Group.Aggregate) {
                 Stream<NumericGroup> result = tx.query().match(query.asMatchGroupAggregate());
-                printCancellableResult(result, x -> printer.numericGroup(x, tx));
+                printCancellableResult(executorService, result, x -> printer.numericGroup(x, tx));
             } else if (query instanceof GraqlCompute) {
                 throw new GraknClientException("Compute query is not yet supported");
             }
         }
+
+        executorService.shutdown();
+        try {
+            if (!executorService.awaitTermination(800, TimeUnit.MILLISECONDS)) {
+                executorService.shutdownNow();
+            }
+        } catch (InterruptedException e) {
+            executorService.shutdownNow();
+        }
```

Comment:
this needed since without shutting down executorService, the console process will hang once you leave it and the console will be unclosable. The value 800 is taken from the example here https://www.baeldung.com/java-executor-service-tutorial#:~:text=To%20properly%20shut%20down%20an,executorService. and doesn't carry any specific meaning to it